process_win.cpp

```diff
@@ -5,9 +5,35 @@
 
 Process::Data::Data(): id(0), handle(NULL) {}
 
+namespace {
+
+// Simple HANDLE wrapper to close it automatically from the destructor.
+class Handle {
+public:
+  Handle() : handle(INVALID_HANDLE_VALUE) { }
+  ~Handle() {
+    close();
+  }
+  void close() {
+    if (handle != INVALID_HANDLE_VALUE)
+      ::CloseHandle(handle);
+  }
+  HANDLE detach() {
+    HANDLE old_handle = handle;
+    handle = nullptr;
```

Comment:
You're right, I forgot to change that line (initially I've created a `Handle` impl using `nullptr` as initial value, and then change it to `INVALID_HANDLE_VALUE`. I'll push a fix for this.